test/html.js

```diff
@@ -510,7 +509,91 @@ describe('HTML Parser', function () {
 				a.removeChild(c);
 				a.childNodes.length.should.eql(1);
 			});
+		});
 
+		describe('#getElementsByTagName', function () {
+			it('find all divs', function () {
```

Comment:
I think we can drop this test and add the for loop check to the next test, as the logic can be a bit redundant.

Reply:
deleted & moved the loop one test ahead